.travis/runsyspycmd.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
+
+# Execute command in environment python only if environment python has access
+# to system python libraries, especially C bindings. The script is run with
+# these arguments:
+#
+#   $1     - path to environment python
+#   $2     - path to system python
+#   $3     - command runnable in Python (should be present in $PATH)
+#   ${@:4} - arguments passed to $3
+
+set -e
+
+ME=$(basename $0)
+SCRIPTDIR=$(readlink -f $(dirname $0))
+TOPDIR=$(readlink -f ${SCRIPTDIR}/..)
+
+. ${SCRIPTDIR}/utils.sh
+. ${SCRIPTDIR}/config.sh
+
+# Run user defined hook from .travis/config.sh.
+lsr_runsyspycmd_hook "$@"
```

Comment:
Is it correct to run the hook before the check for syspython == envpython?

Reply:
It allows to run commands also in a cases when the script normally exits (for example, user can specify alternative actions that should be taken in a case there is no venv python to system python match). On the other hand, it is disputable whether is such a scenario real in our case, so maybe for now we can put the hook away for simplicity?

Reply:
After some consideration, I removed `lsr_runsyspycmd_hook` to make wrapper more cleaner. It has no use for us at now and I doubt about its usefulness in future.